g6/auditInformationProvided.yml

```diff
@@ -3,16 +3,14 @@ filters: 'Consumers are aware of the audit information that will be provided to
 hint: 'Do you provide audit information to consumers? Choose 1.'
 dependsOnLots: 'SaaS, PaaS, IaaS'
 assuranceApproach: 2answers-type1
-fields:
+type: radios
+options:
     -
-        type: radio
         label: None
```

Comment:
Does this need a 'filterLabel' too, or is 'label' is used when 'filterLabel' doesn't exist?  And if that's the case why do we have duplication for other options here? It looks like 'label' and 'filterLabel' have the same value for many questions.